### DISCUSSION TITLE
Announcing mediapipe_helper: A helper package for using MediaPipe's face_mesh package

### DISCUSSION THREAD

**Discussion by LearningnRunning** [google-ai-edge/mediapipe-samples#342] [Show and tell]
# Introduction
The face_mesh package in MediaPipe is a powerful tool for facial landmark detection. However, it can be difficult to use, especially for beginners. The mediapipe_helper package is designed to make it easier to use the face_mesh package.

Features

The mediapipe_helper package provides a number of features that make it easier to use the face_mesh package, including:
- convert_landmark : The convert_landmark result can be used to raise certain parts of the face, or to mark certain parts of the face.
- Finding symmetric indexes: Entering index tells you the index of the landmark that is symmetrical to the center line of the face.
- Landmark list template by facial part: You get a list of landmarks such as eyes, pupils, eyebrows, nose, mouth, and face borders.
- Drawing landmark indices on faces with a list of landmarks

## Installation

To install the mediapipe_helper library, you can use pip:

```bash
pip install mediapipe_helper
```

or

```bash
git clone https://github.com/LearningnRunning/py_face_landmark_helper

python setup.py install
```
You can check out more here: https://github.com/LearningnRunning/py_face_landmark_helper.

 If you have anything to add or modify to this package, please let me know. Thank you.

**Comment by alanwilter** [comment]
Your code is using legacy API AFAIK, though it does work with Linux, but for macOS, only up to 0.10.9.
See https://github.com/google/mediapipe/issues/5168 and https://github.com/google/mediapipe/issues/5188
